syntax/php.vim

```diff
@@ -670,7 +670,23 @@ endif
 syn case match
 
 " HereDoc
-if version >= 704
+if version >= 730
```

Comment:
This will check against Vim's version, try `:echo version` to confirm.

Reply:
Ah, o just guess really I don't have any idea really how to add this. Is there a way to check for the PHP version or should I just update the end search?